apps/antalmanac/src/actions/AppStoreActions.ts

```diff
@@ -86,10 +86,14 @@ export const saveSchedule = async (userID: string, rememberMe: boolean) => {
             try {
                 await trpc.users.saveUserData.mutate({ id: userID, userData: scheduleSaveState });
 
-                openSnackbar(
-                    'success',
-                    `Schedule saved under username "${userID}". Don't forget to sign up for classes on WebReg!`
-                );
+                if (!autoSave) {
+                    openSnackbar(
+                        'success',
+                        `Schedule saved under username "${userID}". Don't forget to sign up for classes on WebReg!`
+                    );
+                } else {
+                    openSnackbar('success', `Updated`);
```

Comment:
I think this could probably be a little less terse, but I'm not too sure what the message *should be*. Open to suggestions!

Reply:
Or maybe not using the snack bar to alert the user at all?
Use the following style for `SAVE` to show that there is a saving going on.

<img width="189" alt="image" src="https://github.com/icssc/AntAlmanac/assets/47913844/cdb8122b-e391-4ff0-8128-9bc3c54fd103">